.github/workflows/update-on-merge.yaml

```diff
@@ -50,7 +50,7 @@ jobs:
       id: renumber-sections
       run: |
         cd tools
-        ./run-section-renumber.sh
+        ./run-section-renumber-update.sh
```

Comment:
Yes, this is a hack workaround. I'll fix when I update the reporting for the word converter and the test runner.